sources/packages/backend/apps/api/src/route-controllers/institution-locations/institution-location.controller.service.ts

```diff
@@ -105,6 +106,9 @@ export class InstitutionLocationControllerService {
       ...transformAddressDetailsForAddressBlockForm(
         institutionLocation.data.address,
       ),
+      isBCPublicInstitution:
+        institutionLocation.institution.institutionType.id ===
+        INSTITUTION_TYPE_BC_PUBLIC,
```

Comment:
Sorry, I am not able to follow why we need this code. 

If it is regarding the AC - `Not available for Public Institutions`, it means BC Public institutions can do a student search in UI and see application and assessment screens of a student application and this AC states that BC Public institutions must not have the new side nav button "Restrictions Management". 

The BC Public institutions **_do not share_** the same side bar as ministry.

Their side bar is `InstitutionApplicationSideBar.vue` so no need of any logic to restrict BC Public View.

Reply:
`isBCPublicInstitution` is removed.